<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PauseDialog</class>
 <widget class="QDialog" name="PauseDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>155</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>128</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>155</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>You are in pause ...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>  Pause Management  </string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="labelPauseStartedOn">
            <property name="text">
             <string>Pause started on </string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelFullPauseTime">
              <property name="text">
               <string>Full pause time of the day</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEditFullPauseTime">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="time">
               <time>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
               </time>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxIncludeExistingOrDefaultPause">
            <property name="text">
             <string>Include  default or existing pause of </string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../EasyTimeTracker.qrc">:/images/tt_2black_coffees_bk_gnd_60x60.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Click me when you are back from your pause</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../EasyTimeTracker.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PauseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
